applications/Tools/quac/manifest.yml

```diff
@@ -0,0 +1,12 @@
+sourcecode:
+  type: git
+  location:
+    origin: https://github.com/rdefeo/quac.git
+    commit_sha: b884fea5a669ba3ca8b3536d9b27df9f54fafce4
+description: "@./README.md"
+changelog: "@./CHANGELOG.md"
```

Comment:
This file does not exist in the specified git revision.

Please fix it and run validation script.

Reply:
Updated the manifest.yml to the latest git revision commit_sha, and re-ran the validation script!

Reply:
So, what was the validation script's output?

I see 
`Error: 09:59:24.031 [E] Icon quac.png is not black and white`

Reply:
Working on changes - I completely missed this error, my apologies. And even though my icon looked fine on the Flipper and in image apps, when I dumped the raw data, the whites were actually 255, 254, 254 <smacks head on desk>

Updating to latest firmware and addressing some other issues I recently found. I appreciate your patience - thank you